packages/dropdowns/src/styled/items/StyledItemMeta.ts

```diff
@@ -23,7 +23,7 @@ export const StyledItemMeta = styled.span.attrs({
   'data-garden-version': PACKAGE_VERSION
 })<IStyledItemMetaProps>`
   display: ${props => (props.isCompact ? 'none' : 'block')};
-  line-height: ${props => props.theme.space.base * 4}px;
+  line-height: ${props => props.theme.lineHeights.sm};
```

Comment:
@rossmoody outside of `typography` we never reference the `lineHeights` theming object. This allows `space.base` to take maximum effect across the component base.